css/style.css

```diff
@@ -93,15 +97,184 @@ flex-wrap: wrap;
     font-weight: 600;
 }
 /***** End navbar *****/
-/***** hero *****/
-.hero{
-    position: relative;
-    height: 100vh;
-    background: linear-gradient(
-        rgba(0, 0, 0, 0.5),
-        rgba(0, 0, 0, 0.5)     
-    ),url(../assets/images/hero-cover.5.png);;
-    background-repeat: no-repeat;
-    background-size: cover;
-}
-/***** End hero *****/
\ No newline at end of file
+    /***** hero *****/
+    .hero-bg{
+        padding-bottom: 2rem;
+        background: linear-gradient(
+            rgba(0, 0, 0, 0.5),
+            rgba(0, 0, 0, 0.5)     
+            ),url(../assets/images/hero-cover.5.png);;
+            background-repeat: no-repeat;
+            background-size: cover;
```

Comment:
"background-size should come after background-position property separated with a slash(/)."
I used the background property with the rest of the attributes except for the "size" because I don't have 'background-position'